topi/python/topi/nn/conv2d.py

```diff
@@ -397,3 +423,13 @@ def conv2d_winograd_without_weight_transform(input, filter, strides, padding,
         4-D with shape [batch, out_height, out_width, out_channel]
     """
     raise ValueError("missing register for topi.nn.conv2d_winograd_without_weight_transform")
+
+def check_skylake(target):
```

Comment:
This doesn't really belong in a generic file like nn/conv2d.py right? Shouldn't this be in some x86/ specific directory?